src/main/kotlin/com/openlattice/chronicle/study/StudyController.kt

```diff
@@ -104,4 +104,35 @@ class StudyController @Inject constructor(
 
     }
 
+    @Timed
+    @DeleteMapping(
+        path = [STUDY_ID_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    override fun destroyStudy(@PathVariable studyId: UUID) {
+        accessCheck(AclKey(studyId), EnumSet.of(Permission.OWNER))
+        logger.info("Deleting study with id $studyId")
+
+        val (flavor, hds) = storageResolver.getPlatformStorage()
+        check(flavor == PostgresFlavor.VANILLA) { "Only vanilla postgres supported for studies." }
+        AuditedOperationBuilder<Unit>(hds.connection, auditingManager)
+            .operation { connection -> studyService.deleteStudies(connection, listOf(studyId)) }
+            .audit {
+                listOf(
```

Comment:
You will actually need to pull this list of organization associated with the study as well and make audit entries for them. The easy way to do this is just load the study (it's pretty light weight) and that will automatically do the left join with organization studies and you can basically re-use logic from create study to make all the necessary audit entries.